sympy/vector/coordsysrect.py

```diff
@@ -170,6 +180,16 @@ def __str__(self, printer=None):
     def __iter__(self):
         return iter([self.i, self.j, self.k])
 
+    def __getattr__(self, name):
```

Comment:
I would be careful with the API choice, especially operator overloading. Is the coordinate system object a dictionary-like object? I don't think so.

The best would be to make the API look like the one in `sympy.diffgeom.CoordSystem`.

https://github.com/sympy/sympy/blob/6ebedba73a61c645a14764bd42e9559f64b56a3f/sympy/diffgeom/diffgeom.py#L298